sdcm/nemesis.py

```diff
@@ -349,6 +349,7 @@ def disrupt_hard_reboot_node(self):
         self.target_node.wait_db_up()
         self.log.info('Waiting JMX services to start after node reboot')
         self.target_node.wait_jmx_up()
+        self.cluster.wait_for_nodes_up_and_normal(nodes=[self.target_node])
```

Comment:
Cql listening port will be ready after node is UP (gossip/nodetool).
But wait_for_nodes_up_and_normal uses nodetool, and it depends on jmx service.
So this order is fine.